### PR TITLE
feat(home): Phase 1 — HomeHero Magazine restyle + state machine copy

### PR DESCRIPTION
**Pull request by iwash-alt**
## Summary

Phase 1 of the home-tab Magazine rework (sub-cycle 3.2 of launch-readiness Phase 3). Rebuilds the home hero on Magazine doctrine per the locked mockup and rewrites the copy for all 5 scenarios.

- `HomeHero.tsx` fully restyled — drops the gold gradient rail + Steel-Page Badge sub-component; replaces with an Archivo tracked-uppercase eyebrow (three tones: brass / olive / dim), italic Bodoni h1 with `<em>` emphasis (sized via the `size: 'm' | 'l'` duo per the locked mockup), italic Bodoni 12.5px subtitle, and a bone-fill primary CTA from `magazine.button.primary`.
- All 5 state-machine builders rewritten — `buildNewUser`, `buildDefault`, `buildDone`, `buildLapsed`, `buildRest` now emit the new copy per spec §7 and use `eyebrow: { text, tone }` (replacing the old `badge: { label, tone }`). `buildDefault` is the sole `size: 'm'` (multi-word stacked headline); the other 4 inherit the `'l'` default. `buildDefault` renders the eyebrow as `Today · Day N of M` by locating `currentDay.id` in the `planDays` rotation.
- `buildLapsed`'s streak object now carries `variant: 'best'` so Phase 2's `StreakStat` can render it as italic-Bodoni-ash (memory, not action). `buildRest` threads an override context (`"Rest counts toward it."`) into `buildStreakStat` instead of using the default.
- `HomeV2Props.streak` gains an optional `variant?: 'current' | 'best'` field — typed in this PR, consumed by `StreakStat.tsx` in Phase 2.
- `HomeStateInput` gains `planDays: WorkoutDay[]` (passed from `plan.days`). `deriveWeekAndDayLabel` (the previous "Week N" helper) is now dead code and was removed.

### Review cycle

Spec-compliance reviewer ran on the initial commit (`e97b738`) and flagged 3 findings:
- **Fix 1 (code)** — HomeHero title size duo (size-m 36px / size-l 42px per locked mockup). Applied in `1e7ee96`; 1 new test added (`expect(props.hero.size).toBe('m')` for default).
- **Fix 2 (originally deferred, then fixed inline)** — eyebrow "Today · Week N" divergence from spec's "Today · Day N of M". Joseph pushed back on the deferral; the data was already in HomeTab's scope, so the fix is small input plumbing not a state-machine restructure. Applied in `c4e7413`; 3 new test assertions; followup issue #412 closed.
- **Fix 3 (docs)** — HomeHero `<section>` aria-label deferred to Phase 2's HomeV2 composition restructure. See "Deferred to Phase 2" below.

## Heightened review discipline

Codex review is OUT OF SCOPE for sub-cycle 3.2 — local review IS the gate. This PR's gate evidence:

- Repo-root vitest sweep: **2165 / 2165 passing** (24.18s)
- Scoped home sweep: **99 / 99 passing** (4 test files)
- `npx tsc --noEmit`: **0 errors**
- `vite build`: clean (pre-existing chunk-size warning ignorable)
- Adversarial 9-class self-review walk completed (documented inline below)
- Visual-diff verification against the locked mockup `.a-hero` block element-by-element
- Dev-server smoke walk: server started cleanly on http://localhost:3000, no compile errors

### Adversarial 9-class self-review

| Class | Status | Notes |
|---|---|---|
| State round-trips | N/A | Pure functional derivation; no state persisted |
| Type casts | Clear | Zero new `as unknown`/`any`/`@ts-ignore` in modified source files. The fixture's `planDays` uses `as WorkoutDay[]` for the test-fixture shape lift — same pattern as `makeDay`/`makeLog`. One `as unknown as { badge?: unknown }` in tests is intentional — locks the badge field is gone from the typed shape. |
| Storage | N/A | No storage code touched |
| Useless defaults | Clear | Eyebrow optional, gated by `{eyebrow && ...}`; subtitle gated the same way as before; `size` defaults to `'l'` at the destructure site. `dayLabel` falls back to `'Today'` for three defensive cases (undefined currentDay, currentDay id not in planDays, empty planDays) — each test-covered. |
| Inert UI | Clear | CTA is `<button type="button">`; eyebrow span has no click handler |
| Date math | Clear | `formatDateHeader` + `daysBetween` unchanged; both use TZ-local arithmetic |
| Hydration | N/A | No `useState`/`useEffect` in HomeHero or the builders |
| setState purity | N/A | No useState anywhere in modified files |
| Browser lifecycle | N/A | Pure presentational; no refs, no listeners |

### Visual-diff against the locked mockup `.a-hero`

Compared element-by-element against `docs/superpowers/mockups/2026-05-19-home-rework-locked.html`:
- Date line: ✓ Archivo tracked uppercase tertiary, margin-bottom
- Eyebrow (brass): ✓ Archivo tracked uppercase brass span (now `Today · Day N of M` per spec §3.2)
- Eyebrow (olive): ✓ Inline-block badge with olive fill + hairline border, rounded
- Eyebrow (dim): ✓ Archivo tracked uppercase tertiary span
- Headline size-l: ✓ Bodoni 42px display, italic on `<em>` (4 scenarios)
- Headline size-m: ✓ Bodoni 36px display, line break before `<em>` for two-line stacked headlines (default scenario)
- Subtitle: ✓ Bodoni italic 12.5px secondary
- CTA: ✓ Bone-fill primary, ground text, full-width

Minor font-size deltas (10px date label vs mockup's 9px) are deliberate per the plan — the React surface scales slightly above mockup-frame sizes for legibility on real device DPRs.

### Dev-server smoke walk

`npm run dev` started cleanly:
- Vite 7.3.1 ready in 224 ms
- Local URL http://localhost:3000
- No compile errors, no console warnings in dev log

I can't actually browse the page in this environment, but the dev server compiled the home feature surface without issue. Joseph should walk the five scenarios in his own browser: newuser (cold-start), default (today is a training day), done (complete a workout), lapsed (>= 7 days idle), rest (today off the plan). Phase 2/3 surfaces (WeekStrip, StreakStat, StatStrip, AtAGlance, NoticedCard, HomeV2 shell) are still Steel Page in this PR — only the hero is restyled.

## Deferred to Phase 2

HomeHero's `<section>` root currently has no aria-label or aria-labelledby. Phase 2
restructures HomeV2 to wrap distinct blocks (week strip, streak, stats, glance, noticed)
in their own `<section>` tags — at that point the hero section will need an explicit name
to disambiguate. Tracking inline; will land with the Phase 2 a11y pass.

## Test plan

- [x] `npx tsc --noEmit` — 0 errors
- [x] `timeout 60 npx vitest run src/features/home/` — 99/99 passing
- [x] `timeout 300 npx vitest run` (repo-root) — 2165/2165 passing
- [x] `node_modules/.bin/vite build` — clean
- [ ] Manual smoke: open dev server, walk newuser / default / done / lapsed / rest scenarios (Joseph)

## Files modified

- `src/features/home/components/HomeHero.tsx` — full rewrite + title size duo
- `src/features/home/homeStateMachine.types.ts` — adds `planDays: WorkoutDay[]` to `HomeStateInput`
- `src/features/home/homeStateMachine.ts` — all 5 builders + `buildStreakStat` override param + `buildDefault` `size: 'm'` + `buildDefault` `Day N of M` computation + `deriveWeekAndDayLabel` removal + period-close on context strings
- `src/features/home/HomeV2.tsx` — adds `streak.variant?: 'current' | 'best'` to the inline type
- `src/features/home/HomeTab.tsx` — passes `plan.days` as `planDays` into the state-machine input
- `src/features/home/homeStateMachine.test.ts` — 45 tests (regression-locks both new + old copy, size assertion, Day N of M assertions including fallbacks)
- `src/features/home/HomeTab.test.tsx` — newuser test asserts `Welcome.` + `First day`; rest test anchored on h1 textContent + dim eyebrow

## Out of scope (Phase 2 + 3)

- `WeekStrip`, `StreakStat`, `StatStrip`, `AtAGlance`, `NoticedCard` sub-components (Phase 2 + 3)
- `HomeV2.tsx` layout shell migration off legacy `color`/`space` barrel (Phase 3)

🤖 Generated with [claude-flow](https://github.com/ruvnet/claude-flow)

### CODE REVIEW

File: src/features/home/homeStateMachine.ts [hunk 1]
```diff
@@ -111,21 +109,20 @@ function buildDefault(input: HomeStateInput): HomeV2Props {
...
-    `${dayExercises.length} ${dayExercises.length === 1 ? 'exercise' : 'exercises'}`,
-    `~${estMinutes} min`,
-  ].filter(Boolean).join(' · ');
+  const exerciseWord = dayExercises.length === 1 ? 'exercise' : 'exercises';
 
   return {
     hero: {
       date: formatDateHeader(input.now),
+      eyebrow: {
+        text: weekAndDay ? `Today · ${weekAndDay}` : 'Today',
```
[the remark this answers is not in the document]
Comment (iwash-alt) [reply src/features/home/homeStateMachine.ts]: Fixed inline on this PR in commit `c4e7413` ("fix(home): Phase 1 — eyebrow renders 'Day N of M' from plan rotation"). Extended `HomeStateInput` with `planDays: WorkoutDay[]` (already in HomeTab scope via `plan.days`), then `buildDefault` computes `dayIndex = planDays.findIndex(d => d.id === currentDay.id)` and renders `"Today · Day ${dayIndex + 1} of ${planDays.length}"`. Defensive fallback to plain `"Today"` for the three edge cases (undefined currentDay, stale id, empty rotation). Removed the now-dead `deriveWeekAndDayLabel` helper. Issue #412 closed (originally opened as a followup; fix landed in the same PR instead). New tests at `homeStateMachine.test.ts` lock the new rendering + a regression-lock asserting the eyebrow does not contain `"Week"`.
